### ISSUE TITLE
Please harmonize the "jump signs"

### ISSUE THREAD

**Issue by jvoisin** [x64dbg/x64dbg#815]
![different jump sizes](http://i.imgur.com/YibAumM.png)

Currently, x64dbg it using `>` and a custom red `v`, it would be great to replace `>` with a red thingy :)

**Comment by mrexodia** [comment]
Also maybe remove those rather pointless dots?

**Comment by wk-952** [comment]
@mrexodia i thought they had different meanings !
isn't the red ones `v` indicate that this instruction will branch, while the black ones `>` indicate that there are references to this instruction ?!

**Comment by mrexodia** [comment]
Yes indeed but this issue refers to the custom drawing of the references
thing.

**Comment by wynick27** [comment]
Currently the indicator has the same meaning as ollydbg, > means jump reference, $ means call reference, . means analyzed(inside a known function).

**Comment by mrexodia** [comment]
Yes but he was referring to custom drawing them instead of displaying text.
